docs/common/install-lcmaps-voms.md

```diff
@@ -0,0 +1,322 @@
+Installing and Maintaining the LCMAPS VOMS Plugin
+=================================================
+
+About This Guide
+----------------
+
+LCMAPS is a software library for mapping grid certificates of incoming connections to specific Unix accounts. The LCMAPS VOMS plugin enables LCMAPS to make mapping decisions based on the VOMS attributes of grid certificates, e.g., `/cms/Role=production/Capability=NULL`. Starting in OSG 3.4, the LCMAPS VOMS plugin will replace GUMS and edg-mkgridmap as the authentication method at OSG sites.
+
+The OSG provides a default set of mappings from VOMS attributes to Unix accounts. By configuring LCMAPS, you can override these mappings, including changing the Unix account that a VO is mapped to, banning based on VOMS attributes, banning a specific user, or adding a VO, VO group, VO role, and/or user that is not in the OSG's set of mappings.
+
+Use this page to learn how to install and configure the LCMAPS VOMS plugin to authenticate users to access your resources on a per-VO basis.
+
+
+Installing the LCMAPS VOMS Plugin
+---------------------------------
+
+Install `lcmaps`, the default mapfile, and the configuration tools:
+
+``` console
+[root@server]# yum install lcmaps vo-client-lcmaps-voms osg-configure-misc
+```
+
+
+Configuring the LCMAPS VOMS Plugin
+----------------------------------
+
+The following section describes the steps required to configure the LCMAPS VOMS plugin for authentication. If you are using OSG 3.3 packages, there are software-specific instructions that must be followed for [HTCondor-CE](Documentation.Release3/InstallHTCondorCE), [GridFTP](Documentation.Release3/InstallOSGGridFTP), and [XRootD](Documentation.Release3/InstallXrootd). To check if you are running OSG 3.3, run the following command:
+
+``` console
+[root@server]# rpm -q --queryformat="%{VERSION}\n" osg-release
+```
+
+Additionally, there is [optional configuration](#OptionalConfig) if you need to make changes to the default mappings.
```

Comment:
To link sections within a page, lowercase the entire section name and replace spaces with dashes. If there are multiple sections with the same name you can link the subsequent sections by appending `_N` where `N` is the section's ordinal number minus one, e.g. append `_1` for the second section. For example, if you have three sections named "Optional Configuration", link them like so:

```
[1st section](#optional-configuration)
[2nd section](#optional-configuration_1)
[3rd section](#optional-configuration_2)
```